lib/oli/delivery/sections.ex

```diff
@@ -2699,6 +2700,60 @@ defmodule Oli.Delivery.Sections do
     end)
   end
 
+  @doc """
+  Maps each resource with its parent container label, being the label (if any) like
+  <Container Label> <Numbering Index>: <Container Title>
+
+  For example:
+
+  %{1: "Unit 1: Basics", 15: nil, 45: "Module 3: Enumerables"}
+  """
+  def map_resources_with_container_labels(section_slug, resource_ids) do
+    containers =
+      from([sr, s, spp, _pr, rev] in DeliveryResolver.section_resource_revisions(section_slug),
+        join: p in Project,
+        on: p.id == spp.project_id,
+        where: s.slug == ^section_slug and rev.resource_type_id == 2,
```

Comment:
WDYT about using Oli.Resources.ResourceType.get_type_by_id/1 instead of the hardcoded id?